.github/CONTRIBUTING.md

```diff
@@ -160,6 +160,7 @@ Issues related: `types`+`contents`+`results`
     - `Irrelevant`: An irrelevant issue for Next
     - `Duplicate`: An issue which had been mentioned
     - `Bug`: A detected bug that needs to be confirmed
+    - `Improvement Need`: An issue that needs to be improvement
```

Comment:
needs to be improved or needs improvements?

Reply:
No matter, it's just for easy finding (Improvement / Improvement Need).